backend/apps/organizations/migrations/0032_auto_20210824_1457.py

```diff
@@ -13,10 +13,10 @@ class Migration(migrations.Migration):
     operations = [
         migrations.RemoveField(
             model_name="organization",
-            name="hr_group",
+            name="admin_group",
```

Comment:
Right, so this is where things get a bit tricky. A while back we had a field on the model that was called `hr_group` and `primary_group`, and we removed them in this migration, instead opting for the methods that you see on the organization model today. 

Django migrations are incredibly powerful, but also a bit magical, so it's tricky to see exactly what's going on. When we go `apps.get_model(...)`, we fetch the organization model as it would look _at that current state_, which actually means that you don't have to make the changes you've made to the migrations prior to and including this one. However: beware that this does _not_ include things such as methods and stuff, _only_ the database-related fields.

Instead, you can simply make the changes that you want in the data migration you've done an absolutely great job of adding, and reduce the scope of the PR a bit.